include/vcpkg/base/util.h

```diff
@@ -142,6 +142,13 @@ namespace vcpkg::Util
         return std::find(begin(cont), end(cont), v);
     }
 
+    template<class Container, class V>
+    auto contains(Container&& cont, V&& v)
+    {
+        using std::end;
+        return find(cont, std::forward<V>(v)) != end(cont);
+    }
+
```

Comment:
```suggestion
    template<class Container, class V>
    bool contains(const Container& cont, const V& v)
    {
        using std::end;
        return Utils::find(cont, v) != end(cont);
    }

```

continuing the same changes as all the other PRs with this function